parm/config/gfs/config.efcs

```diff
@@ -5,17 +5,17 @@
 
 echo "BEGIN: config.efcs"
 
-# TODO: the _ENKF counterparts need to be defined in config.base
-export DO_AERO=${DO_AERO_ENKF:-"NO"}
-export DO_OCN=${DO_OCN_ENKF:-"NO"}
-export DO_ICE=${DO_ICE_ENKF:-"NO"}
-export DO_WAVE=${DO_WAVE_ENKF:-"NO"}
+# TODO: the _ENKF counterparts need to be defined in config.base; for now inherit from config.base
+export DO_AERO=${DO_AERO_ENKF:-${DO_AERO:-"NO"}}
+export DO_OCN=${DO_OCN_ENKF:-${DO_OCN:-"NO"}}
+export DO_ICE=${DO_ICE_ENKF:-${DO_ICE:-"NO"}}
+export DO_WAVE=${DO_WAVE_ENKF:-${DO_WAVE:-"NO"}}
 
 # TODO: Possibly need OCNRES_ENKF, ICERES_ENKF, WAVRES_ENKF too
 if [[ ${DO_OCN} == "YES" ]]; then
   case "${CASE_ENS}" in
     "C48") export OCNRES=500;;
-    "C96") export OCNRES=100;;
+    "C96") export OCNRES=500;;
```

Comment:
See https://github.com/NOAA-EMC/global-workflow/pull/1718/files#r1245789899.

Reply:
This is set because the deterministic and the ensemble share the same ocean resolution, unlike the atmosphere where we have dual resolutions.   I am sure we can support dual resolution ocean as well, but its not worth the hassle at this moment.